### DISCUSSION TITLE
跨库分步式事务 怎么样写

### DISCUSSION THREAD

**Discussion by pengxinglove** [dotnetcore/FreeSql#1522] [Q&A]
作者好！
   我遇到一个事业，涉及到跨库，那采用Freesql应该怎么样写，谢谢！

**Comment by 2881099** [comment]
跨库，要用分布式事务

**Reply by pengxinglove** [reply]
Freesql支持吗，有你的官网上面没有怎么样看到

**Reply by 2881099** [reply]
文档有tcc saga，如果是说transactionscope这个跟adonet没区别

**Comment by d4ilys** [comment]
可以参考下思路 https://github.com/d4ilys/SharingCore
